src/scf-release/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb

```diff
@@ -63,6 +71,13 @@ def run_test
 
             at_exit do
                 set errexit: false do
+                    unless @success
+                        # Print out logs
```

Comment:
Unnecessary comment?

Reply:
It made more sense before I put everything in its own function :D